VehicleCode/Car.cpp

```diff
@@ -1,49 +1,42 @@
 #include "Car.h"
-#include <SD.h> 
```

Comment:
I'm confused why it says you authored this commit because a lot of these changes look like ones I did/would have made

Reply:
I did make this branch off of your car branch so that might be the case?